frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -57,6 +57,7 @@ if Device:setDateTime() then
         sub_item_table = {
           {
             text = _("12 hour clock"),
+            -- does the menu need to be kept open?
```

Comment:
I guess yes - it should allow you to see the change in real time, doesn't it?

Reply:
That's the only reason I left that there, but I wasn't sure if it was necessary to flash the screen, it does look nice though to see the change happen I guess